Tools/LambdaTestTool/src/Amazon.Lambda.TestTool/TestToolStartup.cs

```diff
@@ -267,6 +270,61 @@ private static LambdaFunction LoadLambdaFunction(LambdaConfigInfo configInfo, Lo
             return lambdaFunction;
         }
 
+        private static string DetermineLambdaContext(LocalLambdaOptions localLambdaOptions, CommandLineOptions commandOptions, string lambdaAssemblyDirectory, string lambdaProjectDirectory, RunConfiguration runConfiguration)
+        {
+            var lambdaContext = commandOptions.LambdaContext;
+
+            bool contextFileFound = false;
+            if (!string.IsNullOrEmpty(lambdaContext))
+            {
+                if (Path.IsPathFullyQualified(lambdaContext) && File.Exists(lambdaContext))
+                {
+                    runConfiguration.OutputWriter.WriteLine($"... Using context with from the file {lambdaContext}");
+                    lambdaContext = File.ReadAllText(lambdaContext);
+                    contextFileFound = true;
+                }
+                else
+                {
+                    // Look to see if the context value is a file in
+                    // * Directory with user Lambda assemblies.
+                    // * Lambda project directory
+                    // * Properties directory under the project directory. This is to make it easy to reconcile from the launchSettings.json file.
+                    // * Is a saved sample request from the web interface
+                    var possiblePaths = new[]
```

Comment:
This search path is a duplicate from DeterminePayload. It should be refactored into a utility method and called from both Determine methods.